### PR TITLE
Default: Remove light source from mese ore

### PR DESCRIPTION
**Pull request by paramat**
... New brighter mineral_mese texture

screenshot_20150924_020829

^ New node in a dimly lit cave.

screenshot_20150924_023344

^ Comparison, new node on the lower right with brightness and contrast of texture overlay increased.

See issue #680, fixes the ugly lighting bug shown below and brightens the overlay texture to compensate.

3250d3c6-6082-11e5-9404-53d02caf3bfe

### CODE REVIEW

File: mods/default/nodes.lua [hunk 1]
```diff
@@ -752,11 +752,9 @@ minetest.register_node("default:bronzeblock", {
 minetest.register_node("default:stone_with_mese", {
 	description = "Mese Ore",
 	tiles = {"default_stone.png^default_mineral_mese.png"},
-	paramtype = "light",
```
Comment (kilbith) [inline mods/default/nodes.lua]: Wat !? That's totally unrelated, you'll have a black node in deleting that.

**Comment by kilbith** [comment]
+1.

**Comment by paramat** [comment]
Approval http://irc.minetest.ru/minetest-dev/2015-10-02#i_4412528

**Comment by paramat** [comment]
ac810dbf9d2cdf12f7061dd8d316c8b1214cf7c2

**Pull request closed without merging**